src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+function Navigation() {
+    return <nav className="w-100vw flex-space_between-center px-10 py-1 bg-white" id="nav">
+        <a href="/index.html">
+            <h2>LightsOut</h2>
+        </a>
+        <input type="search" className="input w-40per" placeholder="Search LightsOut" id="search-bar-nav" />
+        <ul className="flex-space_between-center" id="nav-icons">
+            <li className="mx-2">
+                <a href="/Pages/login.html"><button className="btn-primary-confirm">Login</button></a>
```

Comment:
Going forward you can use Link instead.

Reply:
I will rectify the code, thanks for pointing out.